dist/main.js

```diff
@@ -0,0 +1,50 @@
+$('#submitBtn').on('click', () => {
+    const input = $('#teamName').val()
+    $.get(`/teams/${input}`, function (result) {
+        renderer.render(result)
+    })
+})
+
+$('#dreamTeam').on('click', () => {
+    $.get('/dreamTeam', function (result) {
+        renderer.render(result)
+    })
+})
+
+
+$('div').on('click','.addPlayer', function(){
+    const playerFirstName = $(this).data("first")
+    const playerLastName = $(this).data("last")
+    $.ajax({
+        url: `/dreamTeam/${playerLastName}/${playerFirstName}`,
+        method: "POST",
+        success: function(){
+        }
+    })
+})
+
+$('div').on('click','.deletePlayer', function(){
+    const playerFirstName = $(this).data("first")
+    const playerLastName = $(this).data("last")
+    $.ajax({
+        url: `/dreamTeam/${playerLastName}/${playerFirstName}`,
+        method: 'DELETE',
+        success: function(result){
+            renderer.render(result)
+        }
+    })
+
+})
+
+
```

Comment:
You should separate file and concepts, especially when using MVC, write your render functions in a different file